resources/developer-guides/wallet-rpc.md

```diff
@@ -2564,6 +2565,37 @@ $ curl http://localhost:18082/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"c
 ```
 
 
+### **generate_from_keys**
+
+Restores a wallet from a given wallet address, view key, and optional spend key.
+
+Inputs:
+* *restore_height* - integer; (Optional; default = 0) The block height to restore the wallet from.
+* *filename* - string; The wallet's file name on the local file system. 
```

Comment:
"The wallet's file name on the RPC server." or "The wallet's file name." for consistency with other entries and to avoid potential confusion with the user's local file system.